common/djangoapps/util/tests/test_codejail_includes.py

```diff
@@ -0,0 +1,24 @@
+# lint-amnesty, pylint: disable=missing-module-docstring
+
+import unittest
+
+import eia
+import loncapa
+from verifiers import draganddrop
+
+
+class Test_CodeJail_Includes(unittest.TestCase):
```

Comment:
Python convention is to to use "CamelCase" for class names, without underscores between the words.